product-specifications/example-2.md

```diff
@@ -2,12 +2,20 @@
 
 This tool will help users identify animals by asking a series of questions and then provide a list of most likely animals based on the user's responses. This tool will be used by wildlife conservationists to identify animals they encounter.
 
-## User Flow:
+## User Flow
 
-1. Users answer a series of questions about the observed animal.
-2. Based on user inputs, the system will analyze potential matches.
-3. The system will display a list of animals that most closely match the user's responses.
+1. Users answer a series of questions about the observed animal:
+    - What color is the animal?
+    - Where was the animal observed?
+    - How large was the animal?
+3. Based on user inputs, the system will analyze potential matches.
+4. The system will display a list of animals that most closely match the user's responses.
 
-Note: A different team will build the system to perform the analysis in step 2.
+## Animals Included
+- Eagle (Aquila chrysaetos) 🦅
+- Tiger (Panthera tigris) 🐅
+- Giraffe (Giraffa camelopardalis) 🦒
+- Hippo (Hippopotamus amphibius) 🦛
+- Cobra (Ophiophagus hannah) 🐍
 
```

Comment:
Summary:
- Design the UI/UX on figma (1 day)
- Design database schemas for users and animals (1 day)
- Design identification system (ranking algorithm based on user input) - (2.5 days)
- Develop the front-end and back-end (2 days)
- Front-end and Back-end integration (1 day)
- Testing (1 day)
Estimated completion time (8.5 days)